asyncapi/github/service_types.bal

```diff
@@ -49,23 +50,25 @@ public type PullRequestService service object {
     remote function onEdited(PullRequestEvent payload) returns error?;
 };
 
-# Actions related to pull request reviews. The action can be one of submitted, edited or dismissed.
+# Triggers when Choreo recieved a a new event from GitHub related to a GitHub pull request review.
+# Available actions: onSubmitted, onEdited, and onDismissed
 public type PullRequestReviewService service object {
     remote function onSubmitted(PullRequestReviewEvent payload) returns error?;
     remote function onEdited(PullRequestReviewEvent payload) returns error?;
     remote function onDismissed(PullRequestReviewEvent payload) returns error?;
 };
 
-# Actions related to pull request review comments in the pull request's unified diff. 
-# The action can be one of created, edited or deleted. 
+# Triggers when a new event is received from GitHub when a comment is added to a 
+# pull request during a review in the pull request's unified diff view.
```

Comment:
Can't we say like `Triggers when a new event is received from GitHub when a comment is added to a
pull request review's unified diff view.`

Reply:
Thanks a lot for the suggestion. Resolved